### PR TITLE
Flow based test scenario for pgsql snapshot and clone

### PR DESCRIPTION
**Pull request by Akarsha-rai**
Signed-off-by: Akarsha <akrai@redhat.com>

### CODE REVIEW

File: tests/e2e/flowtest/pvc_snapshot_and_clone/test_pgsql_pvc_snapshot_and_clone.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    """
+
+    # @pytest.fixture(autouse=True)
+    # def pgsql_setup(
+    #     self,
+    #     pgsql_factory_fixture,
+    #     multi_snapshot_factory,
+    #     multi_snapshot_restore_factory,
+    #     multi_pvc_clone_factory,
+    # ):
```
Comment (PrasadDesala) [inline tests/e2e/flowtest/pvc_snapshot_and_clone/test_pgsql_pvc_snapshot_and_clone.py:31]: remove commented code

**Review by PrasadDesala — COMMENTED** [review]
LGTM, just one minor comment

File: tests/e2e/flowtest/pvc_snapshot_and_clone/test_pgsql_pvc_snapshot_and_clone.py (hunk 1 above)
Comment (Akarsha-rai) [reply tests/e2e/flowtest/pvc_snapshot_and_clone/test_pgsql_pvc_snapshot_and_clone.py]: Thanks for noticing it, removed now:)

**Review by Shrivaibavi — APPROVED** [review]
LGTM

**Review by jilju — APPROVED** [review]

**Review by PrasadDesala — APPROVED** [review]

**Pull request merged**